db/schema.rb

```diff
@@ -125,6 +125,7 @@
     t.string   "title",                null: false
     t.boolean  "tags_recommendations"
     t.boolean  "tags_measures"
+    t.boolean  "tags_users"
```

Comment:
@tmfrnz This should be done with a rails database migration.

You'd run:
rails g migration AddTagUsersToTaxonomies tag_users:boolean